presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -4158,6 +4159,14 @@ public void testExplainSetSessionWithUsing()
                         "Parameters: [7]");
     }
 
+    @Test
+    public void testExplainAnalyze()
+    {
+        assertExplainAnalyze(
+                noJoinReordering(BROADCAST),
+                "EXPLAIN ANALYZE SELECT * FROM (SELECT nationkey, regionkey FROM nation GROUP BY nationkey, regionkey) a, nation b WHERE a.regionkey = b.regionkey");
```

Comment:
Can you please verify some selected parts of the output?

Like the thing you mentioned in "Advanced join stats are back" comment
https://github.com/prestosql/presto/pull/6115#issuecomment-734507755